exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
+{[String, String|Error]*} cases => {
+  // empty strand
+  ["",                         ""],
+  // single C nucleotide strand
+  ["C",                       "G"],
+  // single G nucleotide strand
+  ["G",                       "C"],
+  // single A nucleotide strand
+  ["A",                       "U"],
+  // single T nucleotide strand
+  ["T",                       "A"],
+  // longer strand
+  ["ACGTGGTCTTAA", "UGCACCAGAAUU"],
+  // strand with invalid nucleotides
+  ["XCGFGGTDTTAA",        Error()],
+  // strand with invalid nucleotides after the first nucleotide
+  ["ACGTFGTBTEAA",        Error()]
+};
+
+test
+parameters(`value cases`)
+void testHamming(String dna, String|Error expected) {
+  if(is String expected) {
+    assertEquals(transcription(dna), expected);
+  } else if(!is Error result = transcription(dna)) {
```

Comment:
I have no ground to stand on for this request other than "all other exercises do it this way", but I've had a space between `if` and the subsequent parenthesis.